bitnami/dataplatform-bp2/values.yaml

```diff
@@ -0,0 +1,518 @@
+  ##
+  wavefront:
+    url: https://YOUR_CLUSTER.wavefront.com
+    token: YOUR_API_TOKEN
+    ## Name of an existing secret containing the token
+    ##
+    existingSecret:
+  ## Wavefront Collector is responsible to get all Kubernetes metrics from your cluster.
+  ## It will capture Kubernetes resources metrics available from the kubelets,
+  ## as well as auto-discovery capabilities.
+  ##
+  collector:
+    ## Rules based discovery configuration
+    ## Ref: https://github.com/wavefrontHQ/wavefront-kubernetes-collector/blob/master/docs/discovery.md
+    ##
+    resources:
+      limits: {}
+      requests:
+        cpu: 200m
+        memory: 10Mi
+    discovery:
+      enabled: true
+      ## Whether to enable runtime discovery configurations
+      ## Ref: https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/blob/master/docs/discovery.md#runtime-configurations
+      ##
+      enableRuntimeConfigs: true
+      ## Can be used to add additional discovery rules
+      ##
+      config:
+      ## auto-discover kafka-exporter
+      ##
+      - name: kafka-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/kafka-exporter*'
+        port: 9308
+        path: /metrics
+        scheme: http
+        prefix: kube.kafka.
+
+      ## auto-discover jmx exporter
+      ##
+      - name: kafka-jmx-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/jmx-exporter*'
+        port: 5556
+        path: /metrics
+        scheme: http
+        prefix: kube.kafkajmx.
+
+      ## auto-discover elasticsearch
+      ##
+      - name: elasticsearch-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/elasticsearch-exporter*'
+        port: 9114
+        path: /metrics
+        scheme: http
+        prefix: kube.elasticsearch.
+
+      ## auto-discover logstash
+      ##
+      - name: logstash-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/logstash-exporter*'
+        port: 9198
+        path: /metrics
+        scheme: http
+        prefix: kube.logstash.
+
+      ## auto-discover spark
+      ##
+      - name: spark-worker-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/spark*'
+        port: 8081
+        path: /metrics/
+        scheme: http
+        prefix: kube.spark.
+
+      ## auto-discover spark
+      ##
+      - name: spark-master-discovery
+        type: prometheus
+        selectors:
+          images:
+          - '*bitnami/spark*'
+        port: 8080
+        path: /metrics/
+        scheme: http
+        prefix: kube.spark.
+  proxy:
+    ## Wavefront Proxy resource requests and limits
+    ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    ##
+    resources:
+      limits: {}
+      requests:
+        cpu: 100m
+        memory: 5Gi
```

Comment:
Please use ["readmenator"](https://github.com/bitnami-labs/readmenator) format for metadata.